src/constants/index.js

```diff
@@ -2,19 +2,19 @@
 
 const { stripIndents } = require('common-tags');
 
-const DEFAULT_HEADING = (
-  owner,
-  repo,
-) => `🚀 *${owner}/${repo} deployed to production!* Related stories:\n _PRs need correct syntax to appear in the list. Problem? Please report it <https://github.com/target/captains-log/issues|here>._
-`;
-
-const EMPTY_MESSAGE = (owner, repo) => stripIndents`\n
-  *No stories linked in this release for ${owner}/${repo}.* 🤷‍♀️ \n
+const DEFAULT_HEADING = ( owner, repo,) => 
+`🚀 *${owner}/${repo} deployed to production!* Related stories:\n`;
 
-_Think something is broken? Report it <https://github.com/target/captains-log/issues|here>._
+const EMPTY_MESSAGE = (owner, repo) => stripIndents
+`🚀 *${owner}/${repo} deployed to production!*\n
+  *No stories linked in this release* 🤷‍♀️ \n
 `;
 
+const DEFAULT_FOOTER = stripIndents
```

Comment:
Hey @benzvan, thanks for the pull request! It is great to see solutions to problems in code, I really appreciate it. 

I like the addition to the `EMPTY_MESSAGE` where we indicate that there was still a deployment to production, there were just nothing to report. 

I believe we can keep all of the changes in this pull request local to this file. It would look something more like this: 

```
const DEFAULT_ISSUE_MESSAGE = stripIndents`_PRs need correct syntax to appear in the list. Problem? Report it <https://github.com/target/captains-log/issues|here>._`;

const DEFAULT_HEADING = ( owner, repo,) => 
`🚀 *${owner}/${repo} deployed to production!* Related stories:\n
${DEFAULT_ISSUE_MESSAGE}
`;

const EMPTY_MESSAGE = (owner, repo) => stripIndents
`🚀 *${owner}/${repo} deployed to production!*\n
  *No stories linked in this release* 🤷‍♀️ \n
${DEFAULT_ISSUE_MESSAGE}
`;
```

And then we can remove the changes to `src/index.js`.